src/Icon/Icon.tsx

```diff
@@ -1093,6 +1093,23 @@ export const Icon: FC<IconProps> = ({
         />
       </svg>
     )}
+    {render === 'clipboard' && (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="16"
+        height="16"
+        fill="none"
+      >
+        <path
+          fill="#000"
```

Comment:
can we change any of these fill value to use the dynamic `theme.colors[color]` instead?

Reply:
makes sense, changed them!